ruby_on_rails-postgres/web/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ruby:2.6.5
+
+# install nodejs / libdev / postgres-client
+RUN apt-get update -qq && apt-get install -y nodejs libpq-dev postgresql-client
+
+RUN mkdir /myapp
+WORKDIR /myapp
+COPY Gemfile /myapp/Gemfile
+COPY Gemfile.lock /myapp/Gemfile.lock
+
+##
+RUN gem install bundler:2.0.2
+RUN bundle install
+COPY . /myapp
+
+## install yarn
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
```

Comment:
The yarn install should be placed before the copy of the Gemfile to benefit of the layering cache.
Here, each time you add a dependency or change a source file, the installation of yarn will be done.
If you move it before the `RUN mkdir /myapp` command , it will be done only if there is a new release of  `nodejs`, `libpq-dev` or `postgresql-client`.